<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-O4CD7HI" failures="2" tests="3" name="Default test(failed)" time="0.054" errors="0" timestamp="2024-10-08T16:15:52 PDT">
  <testcase classname="sakkujava2.DemoTestNG" name="printdata" time="0.01">
    <failure type="java.lang.AssertionError" message="expected [2000] but found [200]">
      <![CDATA[java.lang.AssertionError: expected [2000] but found [200]
at sakkujava2.DemoTestNG.printdata(DemoTestNG.java:15)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- printdata -->
  <testcase classname="sakkujava2.DemoTestNG" name="printdata1" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [HelloO] but found [Hello]">
      <![CDATA[java.lang.AssertionError: expected [HelloO] but found [Hello]
at sakkujava2.DemoTestNG.printdata1(DemoTestNG.java:25)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- printdata1 -->
  <testcase classname="sakkujava2.DemoTestNG" name="printdata2" time="0.0">
    <skipped/>
  </testcase> <!-- printdata2 -->
  <testcase classname="sakkujava2.DemoTestNG" name="printdata3" time="0.0">
    <ignored/>
  </testcase> <!-- printdata3 -->
</testsuite> <!-- Default test(failed) -->
